eng/CodeAnalysis.ruleset

```diff
@@ -1,4 +1,7 @@
 ﻿<RuleSet Name="Microsoft.Analyzers.ManagedCodeAnalysis" Description="Microsoft.Analyzers.ManagedCodeAnalysis" ToolsVersion="14.0">
+  <Rules AnalyzerId="Microsoft.VisualStudio.Threading.Analyzers" RuleNamespace="Microsoft.VisualStudio.Threading.Analyzers">
+    <Rule Id="VSTHRD103" Action="None" />
```

Comment:
This isn't turning on 103... it's using the default levels for all the rules in this library but explicitly turning _off_ 103.

Reply:
Oops, I got it backwards. Thanks for noticing.